client/valor/schemas/__init__.py

```diff
@@ -1,44 +1,110 @@
-from .constraints import (
-    BoolMapper,
-    Constraint,
-    DictionaryMapper,
-    GeometryMapper,
-    GeospatialMapper,
-    LabelMapper,
-    NumericMapper,
-    StringMapper,
-)
 from .evaluation import EvaluationParameters, EvaluationRequest
-from .filters import Filter
-from .geometry import (
-    BasicPolygon,
+from .filters import Constraint, Filter
+from .symbolic.annotations import (
     BoundingBox,
+    BoundingPolygon,
+    Embedding,
+    Raster,
+    Score,
+    TaskTypeEnum,
+)
+from .symbolic.atomics import (
+    Bool,
+    Date,
+    DateTime,
+    Duration,
+    Equatable,
+    Float,
+    Integer,
+    LineString,
+    MultiLineString,
+    MultiPoint,
     MultiPolygon,
+    Nullable,
     Point,
     Polygon,
-    Raster,
+    Quantifiable,
+    Spatial,
+    String,
+    Symbol,
+    Time,
+    Variable,
 )
-from .metadata import dump_metadata, load_metadata, validate_metadata
+from .symbolic.functions import (
+    And,
+    Eq,
+    Ge,
+    Gt,
+    Inside,
+    Intersects,
+    IsNotNull,
+    IsNull,
+    Le,
+    Lt,
+    Ne,
+    Negate,
+    Or,
+    Outside,
+    Xor,
+)
+from .symbolic.static_collections import (
```

Comment:
nit: should we call this "collections", or just move it under "types"?

Reply:
renamed to `collections.py`!